textdb/textdb-textql/src/main/java/edu/uci/ics/textdb/textql/statements/predicates/SelectPredicate.java

```diff
@@ -5,8 +5,8 @@
 /**
  * Object representation of a "SELECT (...)" predicate inside a { @code SelectExtractStatement }.
  * Subclasses have specific fields related to its projection functionalities.
- * SelectPredicate --+ SelectAllPredicate
- *                   + SelectFieldsPredicate
+ * SelectPredicate --+ SelectAllFieldsPredicate
```

Comment:
@zuozhiw In another PR we discussed the idea of making "attribute" and "field" consistent.  If we decide to keep "field", should we ask @FlavioBayer to use this "attribute" name in TextQL?  Please confirm.